packages/command-handler-svc/src/service.ts

```diff
@@ -0,0 +1,423 @@
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ * Arg Software
+ - José Antunes <jose.antunes@arg.software>
+ - Rui Rocha <rui.rocha@arg.software>
+
+ --------------
+ ******/
+
+"use strict";
+
+import {
+	QuotingAggregate,
+	IParticipantService,
+	IQuoteRepo,
+    IBulkQuoteRepo,
+    IAccountLookupService,
+} from "@mojaloop/quoting-bc-domain-lib";
+import {
+    ParticipantAdapter,
+    MongoQuotesRepo,
+    MongoBulkQuotesRepo,
+    AccountLookupAdapter,
+} from "@mojaloop/quoting-bc-implementations-lib";
+import {existsSync} from "fs";
+import express, {Express} from "express";
+import {Server} from "net";
+import {IAuditClient} from "@mojaloop/auditing-bc-public-types-lib";
+import {ILogger, LogLevel} from "@mojaloop/logging-bc-public-types-lib";
+import {
+	AuditClient,
+	KafkaAuditClientDispatcher,
+	LocalAuditClientCryptoProvider
+} from "@mojaloop/auditing-bc-client-lib";
+import {KafkaLogger} from "@mojaloop/logging-bc-client-lib";
+import {
+	MLKafkaJsonConsumer,
+	MLKafkaJsonProducer,
+	MLKafkaJsonConsumerOptions,
+	MLKafkaJsonProducerOptions
+} from "@mojaloop/platform-shared-lib-nodejs-kafka-client-lib";
+import {IMessageConsumer, IMessageProducer} from "@mojaloop/platform-shared-lib-messaging-types-lib";
+import process from "process";
+import {QuotingCommandHandler} from "./handler";
+import {
+	AuthenticatedHttpRequester,
+    LoginHelper
+} from "@mojaloop/security-bc-client-lib";
+import {IAuthenticatedHttpRequester} from "@mojaloop/security-bc-public-types-lib";
+import {IMetrics} from "@mojaloop/platform-shared-lib-observability-types-lib";
+import {PrometheusMetrics} from "@mojaloop/platform-shared-lib-observability-client-lib";
+
+import {IConfigurationClient} from "@mojaloop/platform-configuration-bc-public-types-lib";
+import {DefaultConfigProvider, IConfigProvider} from "@mojaloop/platform-configuration-bc-client-lib";
+import {GetQuotingConfigSet} from "@mojaloop/quoting-bc-config-lib";
+import {     
+    IBulkQuote,
+    IQuote,
+} from "@mojaloop/quoting-bc-public-types-lib";
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const packageJSON = require("../package.json");
+
+const BC_NAME = "quoting-bc";
+const APP_NAME = "command-handler-svc";
+const APP_VERSION = packageJSON.version;
+const PRODUCTION_MODE = process.env["PRODUCTION_MODE"] || false;
+const LOG_LEVEL: LogLevel = process.env["LOG_LEVEL"] as LogLevel || LogLevel.DEBUG;
+
+const KAFKA_URL = process.env["KAFKA_URL"] || "localhost:9092";
+const MONGO_URL = process.env["MONGO_URL"] || "mongodb://root:mongoDbPas42@localhost:27017/";
```

Comment:
Issue reported by sonarcloud with comment: "Make sure this MongoDB database password gets changed and removed from the code."

Full analysis on this PR is here: https://sonarcloud.io/summary/new_code?id=mojaloop_quoting-bc&pullRequest=17